.github/workflows/ci.yaml

```
@@ -176,4 +176,14 @@ jobs:
-      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
\ No newline at end of file
+      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+
+#  # Run dokka
+#  generate-docs:
+#    name: Generate API docs
+#    uses: ./.github/workflows/generate-docs.yaml
+
+  publish-docs:
+    name: Publish API docs
+    uses: ./.github/workflows/publish-docs.yaml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/publish-docs.yaml": open /home/jailuser/git/.github/workflows/publish-docs.yaml: no such file or directory

(workflow-call)